README.md

```diff
@@ -36,6 +36,18 @@ http://layoutmanager.org/From-0.8-to-0.9
 [Full Release
 Log](https://github.com/tbranyen/backbone.layoutmanager/blob/master/CHANGELOG.md)
 
+#### Cheerio Users
+
+Backbone 1.2.* and above are no longer compatible with cheerio out of the box, so be sure to add the following [custom hook](https://github.com/jashkenas/backbone/wiki/Using-Backbone-without-jQuery):
```

Comment:
Oh my bad, I originally assumed `node/index.js` was just for testing purposes, I see now it's the `main` in package.json. I'll push another commit later tonight